src/jumpstart/saga.ts

```diff
@@ -329,6 +331,17 @@ export function* sendJumpstartTransactions(
 
     ValoraAnalytics.track(JumpstartEvents.jumpstart_send_succeeded, trackedProperties)
     yield* put(depositTransactionSucceeded())
+    yield* put(
+      trackPointsEvent({
+        activityId: 'create-live-link',
+        liveLinkType: 'erc20',
+        beneficiaryAddress,
+        transactionHash: jumpstartTxReceipt.transactionHash,
+        networkId,
+        tokenId: sendToken.tokenId,
+        amount: sendAmount,
+      })
+    )
```

Comment:
Something to consider in the future, [to avoid sequential dispatch of actions](https://redux.js.org/style-guide/#avoid-dispatching-many-actions-sequentially), we should listen to the `depositTransactionSucceeded` action (attaching the additional needed data) in the points saga.

Reply:
ah yeah, that's a good callout!

Reply:
oh we had a [similar discussion](https://github.com/valora-inc/wallet/pull/5294/files#r1575826469) with Kathy in the swaps tracker. perhaps we should refactor all tracking to listeners within the points saga to not have this discussion arise over and over again :) i'm sorry i've set the wrong pattern when implementing the first tracking events.

Reply:
See [this PR](https://github.com/valora-inc/wallet/pull/5631) for a potential implementation of this pattern

Reply:
Oh thanks for taking a stab at this 👍